output/plugins/forms/xml/forms.cppcode

```diff
@@ -407,7 +407,26 @@ Written by
 			#ifequal $aui_managed "1"
 			@{ @#include &lt;wx/aui/aui.h&gt; @}
 		</template>
-		<template name="settings">
+        <template name="code_init">
```

Comment:
The name doesn't match the present naming style wich uses the name components in reversed order